cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,12 @@
-/// <reference types="cypress" />
-
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
```

Comment:
Hardcoding URLs in tests can make them brittle and harder to maintain. If the URL changes, you'll have to change it in every place it's used. Instead, consider storing the base URL in a configuration file or environment variable.